templates/resources/fabric_cloud_router.md.tmpl

```diff
@@ -0,0 +1,29 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "equinix_fabric_cloud_router Resource - terraform-provider-equinix"
+subcategory: "Fabric"
+description: |-
+  Fabric V4 API compatible resource allows creation and management of Equinix Fabric Cloud Router
+---
+
+{{/* This template serves as a starting point for documentation generation, and can be customized with hardcoded values and/or doc gen templates.
+
+For example, the {{ .SchemaMarkdown }} template can be used to replace manual schema documentation if descriptions of schema attributes are added in the provider source code. */ -}}
+
+# equinix_fabric_cloud_router (Resource)
+
+Fabric V4 API compatible resource allows creation and management of [Equinix Fabric Cloud Router](https://docs.equinix.com/en-us/Content/Interconnection/FCR/FCR-intro.htm#HowItWorks).
+
+Additional documentation:
+* Getting Started: https://docs.equinix.com/en-us/Content/Interconnection/FCR/FCR-intro.htm#HowItWorks
+* API: https://developer.equinix.com/dev-docs/fabric/api-reference/fabric-v4-apis#fabric-cloud-routers
+
+## Example Usage
+
+Fabric Cloud Router
+{{tffile "examples/resources/equinix_fabric_cloud_router/example_1.tf"}}
```

Comment:
So I don't know how the `resources/fabric_cloud_router.md` got updated correctly based on how you've oriented the examples in this template. The Code shows that example_1.tf is for the marketplace subscription and `example_2.tf` is for the standard cloud router. Could you run this again and see if it shows a diff? I think there's a disconnect here.

Reply:
yes, I will rename example files.